official/mnist/mnist.py

```diff
@@ -143,7 +144,8 @@ def model_fn(features, labels, mode, params):
     return tf.estimator.EstimatorSpec(
         mode=tf.estimator.ModeKeys.TRAIN,
         loss=loss,
-        train_op=optimizer.minimize(loss, tf.train.get_or_create_global_step()))
+        train_op=optimizer.minimize(loss,
+            tf.compat.v1.train.get_or_create_global_step()))
```

Comment:
official/mnist/mnist.py:148: [C0330(bad-continuation), ] Wrong continued indentation (add 24 spaces).

Reply:
@saberkun Thank you for your kind review :) I fixed it.